### PR TITLE
fix #447: [EN] Support BCE, circa and other Dating templates

### LINKED ISSUE #447: [EN] Support circa2 and BCE templates
Also, maybe circa, ante, post
See https://en.wiktionary.org/wiki/Category:Dating_templates

- Wiktionary page: https://en.wiktionary.org/wiki/lest

Wikicode:
```
{{circa2|1200|short=yes}}
```

Output:
```
Circa2
```

Expected:
```
<i>c.<i> 1200
```

---

Model link, if any: https://en.wiktionary.org/wiki/Template:circa2

### PR DESCRIPTION
**Pull request by lasconic**
- [x] Fixes #447
- [x] Tests added/updated.
- [x] `./check.sh` run and eventual issues fixed.
- [x] Updated CONTRIBUTORS.md, if not already done in the past.

### CODE REVIEW

**Comment by lasconic** [comment]
Unit tests are failing because of the <math> tag in "on.wiki". Not sure how we should handle this. I tried something https://github.com/BoboTiG/ebook-reader-dict/pull/525/files#diff-a1fdbb2ac22d9953f6f6ceae3ec4ca02a440357263b4c65376cccb35be286871R124 but apparently it doesn't work...

**Review by BoboTiG — APPROVED** [review]

**Comment by BoboTiG** [comment]
`test_en.py::test_parse_word[Mars]` is failing and needs to be updated ;)

For the other failure, we know it is not because of your changes, so it is OK (and we will need to find a fix later).

**Comment by lasconic** [comment]
Mars should be fixed.

File: tests/test_en.py [hunk 1]
```diff
@@ -121,7 +121,7 @@
                 "Regularly taking (a drug).",
                 "Under the influence of (a drug).",
                 "<i>(mathematics)</i> Having identical domain and codomain.",
-                "<i>(mathematics)</i> Having V^n as domain and <i>V</i> as codomain, for some set <i>V</i> and integer <i>n</i>.",  # noqa
+                "<i>(mathematics)</i> Having V^n as domain and <i>V</i> as codomain, for some set <i>V</i> and integer <i>n</i>."
```
Comment (BoboTiG) [inline tests/test_en.py:124]:
```suggestion
                "<i>(mathematics)</i> Having V^n as domain and <i>V</i> as codomain, for some set <i>V</i> and integer <i>n</i>."  # noqa
```

File: tests/test_en.py [hunk 2]
```diff
@@ -121,7 +121,7 @@
                 "Regularly taking (a drug).",
                 "Under the influence of (a drug).",
                 "<i>(mathematics)</i> Having identical domain and codomain.",
-                "<i>(mathematics)</i> Having V^n as domain and <i>V</i> as codomain, for some set <i>V</i> and integer <i>n</i>.",  # noqa
+                "<i>(mathematics)</i> Having V^n as domain and <i>V</i> as codomain, for some set <i>V</i> and integer <i>n</i>."  # noqa
```
Comment (BoboTiG) [inline tests/test_en.py:124]:
```suggestion
                "<i>(mathematics)</i> Having V^n as domain and <i>V</i> as codomain, for some set <i>V</i> and integer <i>n</i>.",  # noqa
```

**Pull request merged**